ossec_security/assets/dashboards/ossec_sshd.json

```
@@ -0,0 +1,739 @@
+{
+    "title": "OSSEC - SSHD",
+    "description": "-   This dashboard offers comprehensive insights into SSHD events and activities conducted through SSH-like connection attempts, authentication failure.",
```

```suggestion
    "description": "-   This dashboard offers comprehensive insights into SSHD events and activities conducted through SSH-like connection attempts, such as authentication failure.",
```
This is my best guess, but I might be wrong -- how is "authentication failure" related to the rest of the description?

---

Connection attempts and authentication failures are distinct types of events by SSHD.

---

I'll make a new suggestion based on that information, thank you!